scripts/SANS/sans/state/StateObjects/StateAdjustment.py

```diff
@@ -38,8 +38,8 @@ def validate(self):
         if self.calculate_transmission:
             try:
                 self.calculate_transmission.validate()
-            except ValueError as e:
-                is_invalid.update({"StateAdjustment": "The sub-CalculateTransmission state is invalid," " see here {0}".format(str(e))})
+            except ValueError:
+                is_invalid.update({"StateAdjustment": "The sub-CalculateTransmission state is invalid," " see here {str(e)}"})
```

Comment:
This still has `" "`, and the `.format()` was removed without making it an `f`-string